### ISSUE TITLE
"Apperances"/instances model

### ISSUE THREAD

**Issue by parkan** [mediachain/mediachain#127]
How are we modeling the pingbacks/appearances/instances (i.e. distinct situations/context in which an image has appeared)?

I believe this is currently `ArtefactReferenceCell` (first-class chain entry)

This will work for an image with ~dozens of appearances

This will not work for [this image](https://twitter.com/theellenshow/status/440322224407314432?lang=en)

Are we OK with using the `ArtefactReferenceCell` approach until we get to the "good problem to have" viral case?

See also #9

**Comment by vyzo** [comment]
Yeah, I don't think it is appropriate to track all retweets and likes in image propagation with references.
Perhaps we should limit to initial platform appearance, like first X tweets?
We could also think about building a more intelligent tracking mechanism which follows the viral propagation of an artefact and limits references to hubs.

**Comment by bigs** [comment]
i'd say we not track appearances on the chain for the time being. this could exist in a client service until we get a full grip on what the problem domain / use patterns are like.

**Comment by parkan** [comment]
Possible implementations:
- Unbounded collection in separate namespace, defer all scaling concerns
- pubsub channel back to author (not publicly visible)
- g-counter/g-set
- IPNS pointer to counter/capped set (allows GC)
- Some other GC-able primitive
